testsuite/performance/tests/docker-compose.sh

```diff
@@ -370,7 +370,7 @@ case "$OPERATION" in
             crossdc) export DB_CONTAINER=${PROJECT_NAME}_mariadb_dc1_1 ;;
             *) echo "Deployment '$DEPLOYMENT' doesn't support operation '$OPERATION'." ; exit 1 ;;
         esac
-        if [ ! -f "$DATASET_PROPERTIES_FILE" ]; then echo "Operation '$OPERATION' requires a valid DATASET_PROPERTIES_FILE parameter."; exit 1; fi
+        #if [ ! -f "$DATASET_PROPERTIES_FILE" ]; then echo "Operation '$OPERATION' requires a valid DATASET_PROPERTIES_FILE parameter."; exit 1; fi
```

Comment:
Why was this commented out? The next few rows clearly depend on this property and e.g. the `cd` command on line 382 would fail leading to unexpected results.

Reply:
Not sure, seems like it was commented out temporarily and forgotten. Thanks for catching it.